TheFourFifths/consus-client
# [TFF-217] Style main index page

### Summary

I made the main/index page more prettier. I included a font, so don't mind `assets/fonts/`.
The decrease in coverage is due to the lost item functionality being moved to the index page. There are no tests for it because there weren't any tests for it before.

### Checklist


### How to Review

1. Start the app
1. Admire the index page
1. Test the navigation to make I didn't break/forget anything
1. Verify the code is adequate

### Links

- [TFF-217](https://msoese.atlassian.net/browse/TFF-217)
